GitUI/CommitInfo/CommitInfo.cs

```diff
@@ -292,7 +301,7 @@ void StartAsyncDataLoad()
                 {
                     if (_refsOrderDict == null)
                     {
-                        await LoadSortedRefsAsync();
+                        await LoadSortedRefsAsync(cancellationToken);
```

Comment:
My thinking is that this invocation of `LoadSortedRefsAsync` is not needed and should be just removed. 
The other implementation of `LoadSortedRefsAsync` should be the only one and should remain untouched (passing there a cancellation token from here is wrong). To be super safe, the `LoadSortedRefsAsync` should be protected by an instance of the CancellationTokenSequence.
I believe that this extra `LoadSortedRefsAsync` results from an improper conflict solving while merging https://github.com/gitextensions/gitextensions/pull/6332

Reply:
With this call of `await LoadSortedRefsAsync(cancellationToken)` removed at all, `_refsOrderDict` *is* `null` here on startup. It seems to still work, no NRE occurs.

`_refsOrderDict` is `null` even **twice** on startup until it is assigned by the other implementation at line 216. Perhaps this is a starting point for #6520.

Next finding: GetSortedRefs() is not called at all when refreshing FormBrowse (`F5`).
https://github.com/gitextensions/gitextensions/blob/3b015a56b7802c77a3157b080cc0e2cedf0f1851/GitUI/CommitInfo/CommitInfo.cs#L205-L237

Reply:
>Next finding: GetSortedRefs() is not called at all when refreshing FormBrowse (F5).

This is how it has been working from the beginning. We could mark it as "needs to be refreshed" when there is a not found item during the tags/branches sorting phase.